openedx/features/content_type_gating/tests/test_partitions.py

```diff
@@ -0,0 +1,53 @@
+from datetime import date
+import ddt
+from mock import Mock, patch
+
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangolib.testing.utils import CacheIsolationTestCase
+from openedx.features.content_type_gating.partitions import create_content_gating_partition, CONTENT_GATING_PARTITION_ID
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from xmodule.partitions.partitions import UserPartitionError
+
+@ddt.ddt
+class TestContentTypeGatingPartition(CacheIsolationTestCase):
```

Comment:
the tests seem ok to me but frankly i'm not very familiar with partitions.  
if we could create more of an acceptance type test to check for the issue where we broke prod due to the partition being undefined, that may be useful

Reply:
Sorry, which function has those parameters?

Also, the specific instance where prod wasn't working correctly was the second test (where the studio override was on, but config.enabled wasn't).